src/Makefile.am

```diff
@@ -246,6 +247,7 @@ libbitcoin_server_a_SOURCES = \
   iblt.cpp \
   init.cpp \
   dbwrapper.cpp \
+  rest.cpp \
```

Comment:
why `rest.cpp` has been moved up rather than keep it in alphabetical order?

Reply:
I was getting an undefined symbol but turned out link order was not the problem.  I will put it back